tests/test_utils.py

```diff
@@ -131,9 +132,9 @@ def test_get_ssl_cert_file():
     assert get_ca_bundle_from_env() == "tests"
 
     del os.environ["SSL_CERT_DIR"]
-    os.environ["SSL_CERT_FILE"] = "tests/test_utils.py"
+    os.environ["SSL_CERT_FILE"] = str(Path("tests", "test_utils.py"))
     # SSL_CERT_FILE is correctly set, SSL_CERT_DIR is not set.
-    assert get_ca_bundle_from_env() == "tests/test_utils.py"
+    assert get_ca_bundle_from_env() == str(Path("tests", "test_utils.py"))
```

Comment:
Nice catch on these ones. I think the slightly more idiomatic usage would be:

```suggestion
    assert get_ca_bundle_from_env() == str(Path("tests") / "test_utils.py")
```